### ISSUE TITLE
Model with character cards could not read images.

### ISSUE THREAD

**Issue by Naozumi520** [LostRuins/koboldcpp#746]
As title, I loaded a model (Yi 34b) with mmproj, and it's working surprisingly good. However, I encountered an issue when attempting to load the model with a custom character card. The model was unable to accurately identify the elements present in the image, but without character card, everything's good so far.

**Comment by LostRuins** [comment]
Did you enable "AI vision" to "LLaVA" for the image inside the card you want to use?

![image](https://github.com/LostRuins/koboldcpp/assets/39025047/e965e05d-665d-4d79-8d99-a169b04f3a0c)

Also, kindly send a link to the mmproj file you used. As I would like to catalogue all of them.

**Comment by Naozumi520** [comment]
Yes, I'm sure that "AI vision" is enabled by selecting "LLaVA". Not only unable to accurately identify the elements, but it will also saying something doesn't exist.

Also, here is the [mmproj](https://huggingface.co/cjpais/llava-v1.6-34B-gguf/tree/main)

Thanks to your hardwork!

**Comment by LostRuins** [comment]
Please reopen if issues persist
